db.go

```diff
@@ -259,11 +263,11 @@ func (s *ColumnStore) ReplayWALs(ctx context.Context) error {
 	for _, f := range files {
 		databaseName := f.Name()
 		g.Go(func() error {
-			db, err := s.DB(ctx, databaseName)
+			_, err := s.DB(ctx, databaseName)
```

Comment:
nit: add a small comment explaining the side effects of this `DB` call. I don't think it'll be clear to a casual reader.